### PR TITLE
Add Breadcrumbs to navigation

### PR DESCRIPTION
**Pull request by kevthehermit**
As the title says, adds hyperlink 

breadcrumbs for Client, Flows, Collections

breadcrumbs

breadcrumbs3

### CODE REVIEW

File: applications/Rekall/static/js/rekall.js [hunk 1]
```diff
@@ -2123,7 +2123,7 @@ rekall.cell_renderers.collection_ids_renderer = function(
     result += rekall.utils.make_link(
         rekall.globals.controllers.collection_view + "?" + $.param({
           collection_id: collection_ids[i],
-        }),
+        }) + "&" + $.param({client_id: client_id}) + "&" + $.param({flow_id: flow_id}),
```
Comment (scudette) [inline applications/Rekall/static/js/rekall.js]: I think you can just add these to the param() call above.

File: applications/Rekall/views/flows/collection_view.html [hunk 2]
```diff
@@ -2,6 +2,16 @@
 
 <h3 id="title"></h3>
 
+<nav aria-label="breadcrumb" role="navigation">
+  <ol class="breadcrumb">
+      <li class="breadcrumb-item"><a href="{{=URL(c='default', f='index')}}">Home</a></li>
+    <li class="breadcrumb-item"><a href="{{=URL(c='flows', f='inspect_list')}}?client_id={{=client_id}}">{{=client_id}}</a></li>
+      <li class="breadcrumb-item"><a href="{{=URL(c='flows', f='description')}}?client_id={{=client_id}}&flow_id={{=flow_id}}">{{=flow_id}}</a></li>
```
Comment (scudette) [inline applications/Rekall/views/flows/collection_view.html]: You can add these parameters to the var keyword arg of the URL() function:
http://www.web2pyref.com/reference/url-web2py-function

**Review by scudette — CHANGES_REQUESTED** [review]

**Comment by kevthehermit** [comment]
I'm used to Django :) will read the spec and update.

**Comment by scudette** [comment]
This is a cool addition which makes navigation much better! Thanks :-).

The only minor thing is to use the provided functions which do the proper escaping. Using URL() and params() functions escapes parameters according to URL rules (not HTML escaping which is different). So it is always better to let these utility functions construct the URLs.

**Review by scudette — APPROVED** [review]

**Pull request merged**